### PR TITLE
Scan: Remove unused images and replace with icon.

### LINKED ISSUE #2612: Remove unused images and replace with `Icon`

### PR DESCRIPTION
**Pull request by Pitchanai**
Fixed: #2612

## Implementation details
- Remove unused images and replace with icon.

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scan/src/components/NavBar.re [hunk 1]
```diff
@@ -64,7 +64,7 @@ module RenderMobile = {
     let menu = style([width(`px(20))]);
     let twitterLogo = style([width(`px(19))]);
     let telegramLogo = style([width(`px(17))]);
```
Comment (evilpeach) [inline scan/src/components/NavBar.re]: remove unused styles

File: scan/src/components/Header.re [hunk 2]
```diff
@@ -75,14 +75,14 @@ module DesktopRender = {
                     target="_blank"
                     rel="noopener"
                     className=Styles.socialLink>
-                    <img src=Images.twitterLogo className=Styles.twitterLogo />
+                    <Icon name="fab fa-twitter" color=Colors.bandBlue size=16 />
```
Comment (evilpeach) [inline scan/src/components/Header.re]: Don't forget remove the unused style above.

File: scan/src/components/Header.re [hunk 3]
```diff
@@ -75,14 +75,14 @@ module DesktopRender = {
...
                   </a>
                   <a
                     href="https://t.me/bandprotocol"
                     target="_blank"
                     rel="noopener"
                     className=Styles.socialLink>
-                    <img src=Images.telegramLogo className=Styles.telegramLogo />
```
Comment (evilpeach) [inline scan/src/components/Header.re:85]: Here as well.

File: scan/src/components/ValidatorsTable.re [hunk 4]
```diff
@@ -306,11 +308,11 @@ module SortableTHead = {
       />
       <HSpacing size=Spacing.xs />
       {if (sortedBy == asc) {
-         <img src=Images.sortDown className={Styles.downIcon(false)} />;
```
Comment (evilpeach) [inline scan/src/components/ValidatorsTable.re:309]: remove unused styles

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Comment by pzshine** [comment]
image

Can you wrap these social media with `<div>` and align center to them?

**Comment by evilpeach** [comment]
@Pitchanai Please check the integration test.

**Review by pzshine — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Pull request merged**